README.zh-Hans.md

```
@@ -0,0 +1,658 @@
+
+请注意，此列表*不用于*为贡献者提供认可；它仅用于管理 PR 的审查。
+
+类型定义所有者会被同步到文件 [.github/CODEOWNERS](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/.github/CODEOWNERS)，每周更新一次。此数组就是我们的事实来源（Source of Truth）。
+
+## Definitely Typed 历史
+
+Definitely Typed 是 GitHub 上最活跃的软件源之一。你可能想知道这个项目是如何诞生的。@johnnyreilly 整理了 Definitely Typed 的历史。它讲述了 Definitely Typed 早期的故事，从 @borisyankov 创建仓库到成为 TypeScript 生态系统的关键部分。你可以在这里[阅读 Definitely Typed 的故事](https://johnnyreilly.com/definitely-typed-the-movie)。
+
+## 常见问题（FAQ）
+
+#### 这个仓库和 npm 上的 `@types` 软件包究竟是什么关系？
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)